### PR TITLE
KAIZ-89

### PR DESCRIPTION
**Pull request by BrandonC123**
Notes:
Tested POST & GET requests to FaunaDB. Both requests work from frontend. Schedule meeting card uses service layer to create a new meeting using placeholder attendee data (TODO after user login integration).
*May need another field in schema for timezone

### CODE REVIEW

**Comment by ddenaga** [comment]
The `MeetingService` in `frontend/services/meetingService.ts` is not being used. @jasjot1 can you work on this branch with @BrandonC123 ? The parameters for the `createMeeting` function aren't defined, making it a bit difficult to use.

File: frontend/services/meetingService.ts [hunk 1]
```diff
@@ -10,7 +17,7 @@ export async function fetchMeeting(id: string) {
 		console.error('Error fetching meeting:', error.value)
 		return null
 	}
-
+	// @ts-ignore
```
Comment (ddenaga) [inline frontend/services/meetingService.ts:20]: Specify the type in `useFetch<T>` to give it a proper return type.

**Review by ddenaga — APPROVED** [review]
Given that the Fauna schema will most likely change within the next couple sprints, these changes are fine. We'll add tasks to revamp our Fauna stuff.

**Review by bdlowery — APPROVED** [review]
LGTM

**Pull request merged**